packages/ingest/test/test-sftp.js

```diff
@@ -62,25 +51,20 @@ test.after.always(async () => {
   ]);
 });
 
-test('connect and retrieve list of pdrs', async (t) => {
-  const jsftpSpy = sinon.spy(JSFtp);
-  const { sftpMixin } = proxyquire('../sftp', {
-    jsftp: jsftpSpy
-  });
+test.serial('connect and retrieve list of pdrs', async (t) => {
```

Comment:
Why did the tests have to become `.serial`?

Reply:
Running them in serial seemed to cut the runtime down from about 5 seconds to about 4 seconds.